### ISSUE TITLE
<tab> messes up the output of the input if there is multiple completion options

### ISSUE THREAD

**Issue by nomike** [zakaziko99/agnosterzak-ohmyzsh-theme#21]
You can do the following to have the output messed up:
```
 ♥ 10000%  Don  2 Nov - 23:57  ~ 
 @nomike  cd ~
 ♥ 10000%  Don  2 Nov - 23:57  ~ 
 @nomike  cd /s
```
If you now press the <tab>-key this happens:
```
 ♥ 10000%  Don  2 Nov - 23:57  ~ 
 @nomike  cd /s           cd /s
sbin/  snap/  srv/   sys/ 
```
Screenshot:
![image](https://user-images.githubusercontent.com/1063307/32354434-f9b010f4-c029-11e7-9b6a-eb4b626d9797.png)

### Basic Info

- **OS:** GNU/Linux
- **Distribution:** Ubuntu 17.10 (artful)
- **Architecture:** amd64
- **zsh-version:** zsh 5.2 (x86_64-ubuntu-linux-gnu)

**Comment by zakaziko99** [comment]
yes, I noticed this bug. Unfortunately, I can't find a solution for this issue.
I will still be trying.

Thank you @nomike.

**Comment by pavulonx** [comment]
#25 fixes this
Could you review changes @zakaziko99?
